setup.cfg

```diff
@@ -91,6 +91,6 @@ development =
     pytzdata
 pandas =
     pandas>=1.0.0,<1.6.0
-    pyarrow>=8.0.0,<8.1.0
+    pyarrow>=10.0.1,<10.1.0
```

Comment:
Is there any particular reason multiple versions of arrow can't be supported?

Reply:
With Python 3.11 only 10.0.1 and above is officially supported with prebuilt wheels.

Reply:
Major `pyarrow` version can change the ABI they use, which makes then not compatible with each other. Unfortunately; as long as we compile against `pyarrow` we can only support 1 major versions at a time.